src/views/Clients/ViewClients/index.jsx

```diff
@@ -31,9 +36,28 @@ import clientsQuery from './clients.graphql';
 }))
 export default class ViewWorker extends PureComponent {
   state = {
-    clientSearch: '',
+    clientSearch: this.props.user ? this.props.user.credentials.clientId : '',
```

Comment:
Could you rename the class from `ViewWorker` to `ViewClients`, please? It must have been a typo from the past :)